src/openzeppelin/tests/access/test_dual_ownable.cairo

```diff
@@ -55,16 +57,18 @@ fn setup_camel() -> (DualCaseOwnable, IOwnableCamelDispatcher) {
 }
 
 fn setup_non_ownable() -> DualCaseOwnable {
-    let calldata = ArrayTrait::new();
-    set_caller_address(OWNER());
+    let mut calldata = ArrayTrait::new();
+    calldata.append_serde(OWNER());
     let target = utils::deploy(NonImplementingMock::TEST_CLASS_HASH, calldata);
     DualCaseOwnable { contract_address: target }
 }
 
 fn setup_ownable_panic() -> (DualCaseOwnable, DualCaseOwnable) {
-    set_caller_address(OWNER());
-    let snake_target = utils::deploy(SnakeOwnablePanicMock::TEST_CLASS_HASH, ArrayTrait::new());
-    let camel_target = utils::deploy(CamelOwnablePanicMock::TEST_CLASS_HASH, ArrayTrait::new());
+    let mut calldata = ArrayTrait::new();
+    calldata.append_serde(OWNER());
```

Comment:
```suggestion
```
Same here. What's interesting is that this doesn't trigger a panic

Reply:
mmm, right. Is not reverting because calldata check for not-existent constructor is not implemented in the [cairo-lang-runner](https://github.com/starkware-libs/cairo/blob/1edcc130fe701820016f9f4af754543b16bfbf67/crates/cairo-lang-runner/src/casm_run/mod.rs#L863). Is implemented in [blockifier](https://github.com/starkware-libs/blockifier/blob/4af8928fdb82a8a5973595db9da5cee017afc7e7/crates/blockifier/src/execution/entry_point.rs#L367) though.

Reply:
Ah-ha! Good to know, thank you